src/webprojects/relay.md

```diff
@@ -0,0 +1,80 @@
+# Firefox Relay: how to localize and test
+
+Firefox Relay⁩ email masks protect your true email address from public view, automatically forwarding messages to your true inbox. Now you can receive only the messages you want in your inbox. For Canada and the US, phone mask feature as a premium service is offered.
+
+Firefox Relay consists of two parts:
+* [Firefox Relay Website](https://pontoon.mozilla.org/projects/firefox-relay-website)
+* [Firefox Relay Add-on](https://pontoon.mozilla.org/projects/firefox-relay-add-on)
+
+## Firefox Relay Website
+
+This project is open to localization in all languages. The free version with a limit of 5 email masks is available to all users. However, Relay Premium, a paid service, is available in [these markets](https://docs.google.com/document/d/1ww1ab5Vhr4AISMlWZvTe0cnz8GD16F8xE5vegcJZNwQ/edit#) at the moment. The team will continue market expansion as well as adding the product features and making improvements.
+
+Phone masking, a paid service, is offered to limited markets. Localizable content is limited in languages where the markets are offered the Relay Premium services.
+
+* Production: https://relay.firefox.com/
+* Stage: https://fx-relay-demo.netlify.app
+
+**NOTE**:
+* Some locales have more files than other locales. This is due to feature availability and avoiding using community resources for features not available in the mid term or not at all.
+* The staging server is not ideal. It syncs once a day. Premium features are not available for testing. Not all free service features can be tested.
+
+### Localizing the strings
+
+Firefox Relay is written in Fluent, or .ftl format. It contains variables and placeables. Sometimes it contains special punctuations or tags. Make sure to preserve them in the localized strings. For better context, many strings are provided with comments, which can be found above the area where you enter translated strings. If you have additional questions:
+
+* In Pontoon: Ping the l10n PM in charge in the comment field on the right side of the translation field.
+* Matrix channel: Ping the l10n PM in charge with any issues you may have.
+* Localization on Discourse: Follow announcements and discussions, and raise questions there.
+
+### To Test
+
+Firefox Relay Website is synced once a day between localized and the core repo. To test your localization, you need to download the build at https://github.com/mozilla/fx-private-relay/releases. Click on the Assets arrow, you should see two files. Download the appropriate compressed file to your machine. To download the latest l10n build, click on the Tags button (next to the Release button), the latest build is on the top.
```

Comment:
I tried downloading the release build according to these instructions, but wasn't able to successfully build the project. These instructions aren't clear and require troubleshooting to get the project built. Since the translations for the locales are stored in a separate git and built into the project as a submodule, the translation data is not included in the release data. If I'm missing something here, I'd recommend adding more instructions here so localizers can build the project successfully.

Ultimately, I found following the instructions from the git [README](https://github.com/mozilla/fx-private-relay) lead to success, so the other option could be to point localizers to following those instructions. However, that means they wouldn't be building the "current" build.

Reply:
I pointed out before that these instructions don't make sense for the website, these are relevant for the add-on (if integrated).

And they should be removed from Pontoon as well: https://pontoon.mozilla.org/projects/firefox-relay-website/info/